packages/telemetry/src/events/session.ts

```diff
@@ -89,18 +103,24 @@ export function eventCliSession(
 		viteVersion: getViteVersion(),
 		nodeVersion: process.version.replace(/^v?/, ''),
 		// Config Values
-		config: astroConfig
+		config: userConfig
 			? {
-					hasViteConfig: Object.keys(astroConfig?.vite).length > 0,
 					markdownPlugins: [
-						astroConfig?.markdown?.remarkPlugins ?? [],
-						astroConfig?.markdown?.rehypePlugins ?? [],
+						userConfig?.markdown?.remarkPlugins ?? [],
+						userConfig?.markdown?.rehypePlugins ?? [],
 					].flat(1),
-					hasBase: astroConfig?.base !== '/',
-					viteKeys: viteConfigKeys(astroConfig?.vite, ''),
-					adapter: astroConfig?.adapter?.name ?? null,
-					integrations: astroConfig?.integrations?.map((i: any) => i.name) ?? [],
-					experimentalFeatures: getExperimentalFeatures(astroConfig) ?? [],
+					configKeys: configKeys(userConfig, ''),
+					adapter: userConfig?.adapter?.name ?? null,
+					integrations: userConfig?.integrations?.map((i: any) => i.name) ?? [],
+					experimentalFeatures: getExperimentalFeatures(userConfig) ?? [],
+					trailingSlash: userConfig?.trailingSlash,
+					build: userConfig?.build ? {
+						format: userConfig?.build?.format
+					} : undefined,
+					markdown: userConfig?.markdown ? {
```

Comment:
This is already under a `config` object so the `configX` naming seems redundant, but I think I understand what you're getting at.... These extra fields are enum values that I thought we'd want to have inside on. So I guess it could be `config.keys` for the keys and `config.enums` for these enum things..

Reply:
oh doh, actually I think I missed the `config` top-level thing. Ignore me, I think I like `configKeys` and `config` better :)